ENIGMAsystem/SHELL/Graphics_Systems/None/NONEStd.h

```diff
@@ -0,0 +1,94 @@
+/** Copyright (C) 2017 Faissal I. Bensefia
+*** Copyright (C) 2008-2013 Robert B. Colton, Adriano Tumminelli
+*** Copyright (C) 2014 Seth N. Hetu
+***
+*** This file is a part of the ENIGMA Development Environment.
+***
+*** ENIGMA is free software: you can redistribute it and/or modify it under the
+*** terms of the GNU General Public License as published by the Free Software
+*** Foundation, version 3 of the license or any later version.
+***
+*** This application and its source code is distributed AS-IS, WITHOUT ANY
+*** WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+*** FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+*** details.
+***
+*** You should have received a copy of the GNU General Public License along
+*** with this code. If not, see <http://www.gnu.org/licenses/>
+**/
+
+#include <string>
+#include <vector>
+#include "../General/GScolors.h"
+#include "../General/GSprimitives.h"
+#include "../General/GSd3d.h"
+#include "../General/GSstdraw.h"
+#include "../General/GSblend.h"
+#include "../General/GSsurface.h"
+#include "../General/GSscreen.h"
+#include "Universal_System/scalar.h"
+
+namespace enigma
+{
+  extern unsigned char currentcolor[4];
+
+  extern int currentblendmode[2];
+  extern int currentblendtype;
+  extern bool glew_isgo;
+  extern bool pbo_isgo;
+	unsigned get_texture(int texid);
+	void graphics_samplers_apply();
```

Comment:
Is something calling this from outside Graphics_Systems? If this declaration is needed, it probably belongs in either the Bridges folder or the General folder. This applies to a lot of the functions in these systems; they exist because they're important for emulating GM's API in systems with very different implementations, but this system doesn't have an actual implementation.

Reply:
No nothing's calling it, I've removed it as such.